etna/records/views/choosers.py

```diff
@@ -17,21 +19,32 @@ class KongModelChooserMixinIn(ChooserMixin):
     """
 
     # Model belonging to this chooser, set via <model>ChooserViewSet.
-    model = None
+    model: Record = None
 
     # Allow models to be searched via chooser. Hides the search box if False
     is_searchable = True
 
     def get_object_list(self, search_term=""):
```

Comment:
Maybe this is redundant?

Reply:
Thanks @mozaki1, you're right!

Pushed an update to remove.